panel/models/anywidget_component.ts

```diff
@@ -0,0 +1,83 @@
+import type * as p from "@bokehjs/core/properties"
+
+import {ReactComponent, ReactComponentView} from "./react_component"
+
+const panel_adapter = `
+class PanelModel {
+
+  constructor(model) {
+    this.model = model;
+  }
+
+  get(name) {
+    let value
+    if (name in this.model.attributes) {
+      value = this.model[name]
+    } else {
+      value = this.model.data[name]
+    }
+    if (value instanceof ArrayBuffer) {
+      value = new Uint8Array(value)
+    }
+    return value
+  }
+
+  set(name, value) {
+    return this.data[name] = value;
+  }
+
+  save_changes() {
```

Comment:
Would there be cases where the `AnyWidget` esm implementation actually relies on not updating the model before changes are saved?

We are deviating from `AnyWidget` api here which could cause unexpected problems?

Reply:
Don't think we have much of a choice here, at least for now.

Reply:
Nvm, was a simple change.